torchtune/datasets/_chat.py

```diff
@@ -28,11 +28,12 @@ class ChatDataset(Dataset):
     The general flow from loading a sample to tokenized prompt is:
     load sample -> apply transform -> foreach turn{format into template -> tokenize}
 
-    If the column/key names differ from the expected names in the `ChatFormat`,
-    then the `column_map` argument can be used to provide this mapping.
+    If the column/key names differ from the expected names in the ``ChatFormat``,
+    then the ``column_map`` argument can be used to provide this mapping.
+
+    Use ``convert_to_messages`` to prepare your dataset into the llama conversation format
+    and roles::
 
-    Use `convert_to_messages` to prepare your dataset into the llama conversation format
-    and roles:
         [
             {
```

Comment:
while you're here, can you also update this format example? It should actually be something like:
```
[
    Message(
        role=<system|user|assistant>,
        content=<message>,
    ),
    ...
]
```

Reply:
Wait so the user has to create a `Message` class? They can't just have a JSONL file that parses this into a Message format?

Reply:
hm yeah, that would be more ideal... maybe we should add this step in the dataset class to parse it into Message format for you if needed

Reply:
Well we *could* enforce it here on the class, but in the builder, it will format for you?

Reply:
yeah if you specify a converter... need to think more on this, let's revisit this.

Reply:
Definitely, but for now, what should the docstring read?